CDP4ShellDialogs/ViewModels/ModelOpeningDialogViewModel.cs

```diff
@@ -76,25 +118,88 @@ public string DialogTitle
         }
 
         /// <summary>
-        /// Gets the list of <see cref="BaseRowViewModel"/> available
+        /// Gets the title of the group
+        /// </summary>
+        public string ScreenTitle
+        {
+            get { return this.screenTitle; }
+            set { this.RaiseAndSetIfChanged(ref this.screenTitle, value); }
+        }
+
+        /// <summary>
+        /// Gets if model screen is selected
+        /// </summary>
+        public bool IsModelScreen
+        {
+            get { return this.isModelScreen; }
+            set
+            {
+                this.IsIterationScreen = !value;
```

Comment:
We usually do not do custom code in the setter. Instead use a subscription to `this.WhenAnyValue(...)` to react to changes to `IsModelScreen`.